agr_indexer/src/main/java/org/alliancegenome/indexer/Main2.java

```diff
@@ -0,0 +1,36 @@
+package org.alliancegenome.indexer;
+
+import java.util.HashMap;
+
+import org.alliancegenome.core.config.ConfigHelper;
+import org.alliancegenome.curation_api.model.entities.AGMDiseaseAnnotation;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.indexer.indexers.curation.interfaces.AGMDiseaseAnnotationInterface;
+
+import si.mazi.rescu.RestProxyFactory;
+
+public class Main2 {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.
<!-- __reviewdog__:ChA1ZjUyOGIyMDlhNjdhYTgxEgpjaGVja3N0eWxl -->